### ISSUE TITLE
Workflow package will throw this warning about software dependencies

### ISSUE THREAD

**Issue by mikelove** [Bioconductor/BiocCheck#261]
Another warning that looks like a FP for a workflow is:

```
    * Checking for Bioconductor software dependencies...
    ! WARNING: No Bioconductor dependencies detected. Note that some infrastructure
    packages may not have Bioconductor dependencies. For more information, reach
    out to the Bioconductor community and/or consider a CRAN submission.
```

maybe workflows could be detected with BiocType during BiocCheck
